apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.0.0__create_tables.sql

```diff
@@ -12,15 +12,16 @@ CREATE TABLE `challenge_platform`
 );
 
 
--- challenge_input_data_type definition
-CREATE TABLE `challenge_input_data_type`
+-- challenge_data_edam definition
+
+CREATE TABLE `edam_ontology_term`
```

Comment:
Shall we rename this table `edam_term` since you renamed the file `edam_terms.csv`? Actually, do we maintain a convention between the name of the CSV files and their corresponding table names?

Reply:
> do we maintain a convention between the name of the CSV files and their corresponding table names?

No, we do not.  I am not picky though - we can update the table name if you like. My reasoning was that `edam_ontology_term` is more descriptive than `edam_term`, which I know you prefer.

Reply:
I will leave the CSV filenames up to you.